galaxykubeman/values.yaml

```diff
@@ -7,6 +7,28 @@ configs: {}
 secrets: {}
   # db-password: changeme
 
+persistence: {}
+  nfs:
+    name: "nfs-disk"
```

Comment:
what's the difference between `name` in this line and `gcePersistentDisk.pdName` in line 19?

Reply:
oh is this `name` for the PVC/PV pair

Reply:
Yes, the name is used for the k8s resources, which I suppose could just be replaced by the key itself